nautobot/docs/configuration/required-settings.md

```diff
@@ -270,6 +273,24 @@ The following environment variables may also be set for some of the above values
 
 For more details on configuring RQ, please see the documentation for [Django RQ installation](https://github.com/rq/django-rq#installation).
 
+### Task Queuing with Celery
+
+Out of the box you do not need to make any changes to utilize task queueing with Celery. All of the default settings are sufficient for most installations.
+
+In the event you do need to make customizations to how Celery interacts with the message broker such as for more advanced clustered deployments, the following settings are required.
+
+#### CELERY_BROKER_URL
+
+This setting tells Celery and its workers how and where to communicate with the message broker. The default value for this points to ``redis://localhost:6379/0`. Please see the [optional settings documentation for `CELERY_BROKER_URL`](../optional-settings#celery_broker_url) for more information on customizing this setting.
+
+#### CELERY_RESULT_BACKEND
+
+This setting tells Celery and its workers how and where to store message results. This defaults to the same value as `CELERY_BROKER_URL`. In some more advanced setups it may be required for these to be separate locations, however in our configuration guides these are always the same. Please see the [optional settings documentation for `CELERY_RESULT_BACKEND`](../optional-settings#celery_result_backend)) for more information on customizing this setting.
```

Comment:
```suggestion
This setting tells Celery and its workers how and where to store message results. This defaults to the same value as `CELERY_BROKER_URL`. In some more advanced setups it may be required for these to be separate locations, however in our configuration guides these are always the same. Please see the [optional settings documentation for `CELERY_RESULT_BACKEND`](../optional-settings#celery_result_backend) for more information on customizing this setting.
```